package/Resources/Translations/zh-Hans/Translation.ini

```
@@ -0,0 +1,1704 @@
+Client:DTAConfig:Uninstall=卸载; Uninstall
+Client:DTAConfig:Unpacking=解压中...; Unpacking...
+Client:DTAConfig:Update=更新; Update
+Client:DTAConfig:UpdateConfirmRequiredText=要启用 {0}，客户端需要在您的游戏目录中下载必要的文件。@@这将额外占用 {1} 的磁盘空间，下载可能需要一些时间，具体取决于您的网络连接速度。下载大小为 {2}。@@下载期间您将无法玩游戏。您希望继续吗？; To enable {0} the Client will need to download the necessary files to your game directory.@@This will take an additional {1} of disk space, and the download may take some time@depending on your Internet connection speed. The size of the download is {2}.@@You will not be able to play during the download. Do you wish to continue?
+Client:DTAConfig:UpdateConfirmRequiredTitle=需要确认; Confirmation Required
+Client:DTAConfig:VoiceVolume=语音音量：; Voice Volume:
+Client:DTAConfig:WindowedMode=窗口模式; Windowed Mode
+Client:DTAConfig:WriteGameRegistry=将游戏安装路径写入 Windows@注册表 (使其可以加入 CnCNet 上@其他游戏的房间); Write game installation path to Windows@Registry (makes it possible to join@other games' game rooms on CnCNet)
+Client:DTAConfig:YRShowHidden=显示隐藏物体; Show Hidden Objects
+Client:Main:AddFavorite=添加收藏; Add Favorite
+Client:Main:AddFriend=添加好友; Add Friend
+Client:Main:AdminRequiredExplanation=您似乎正在从一个受写入保护的目录中运行 {0}。@@为了让 {1} 在受写入保护的目录中正常运行，需要管理员权限。@@也请确保您的杀毒软件没有阻止 {1}。; You seem to be running {0} from a write-protected directory.@@For {1} to function properly when run from a write-protected directory, it needs administrative priveleges.@@Please also make sure that your security software isn't blocking {1}.
+Client:Main:AdminRequiredRestartPrompt=您想以管理员权限重启客户端吗？; Would you like to restart the client with administrative rights?
+Client:Main:AdminRequiredTitle=需要管理员权限; Administrative privileges required
+Client:Main:AdminSuffix=(管理员); (Admin)
+Client:Main:AdvancedOptions=高级选项; Advanced Options
+Client:Main:AISpectatorsNotification=AI玩家不喜欢旁观，他们渴望玩真的！; AI players don't enjoy spectating matches. They want some action!
+Client:Main:AllGameModes=全部; All
+Client:Main:AllPlayersTab=所有玩家; All Players
+Client:Main:AttemptConnectToServer=正在尝试连接到 {0}; Attempting connection to {0}
+Client:Main:AttemptJoin=正在尝试加入游戏 {0} ...; Attempting to join game {0} ...
+Client:Main:AuthorBy=作者：; By
+Client:Main:AuthorByUnknown=作者未知; By Unknown Author
+Client:Main:AutoAllyAsANoun=自动结盟; auto ally
+Client:Main:AutoAllyingPrefix=自动结盟：; Auto Allying:
+Client:Main:AutoAllyingText1=自动结盟允许主机将起始位置分配给队伍，而不是玩家。当玩家被分配到出生点时，他们将根据这些映射自动分配到队伍。这最适合与随机队伍和随机起点一起使用。但是，只需要随机队伍。手动指定的起点将优先。; Auto allying allows the host to assign starting locations to teams, not players.@When players are assigned to spawn locations, they will be auto assigned to teams based on these mappings.@This is best used with random teams and random starts. However, only random teams is required.@Manually specified starts will take precedence.
+Client:Main:AutoAllyingTextNoTeam=禁止此位置分配给玩家。; Block this location from being assigned to a player.
+Client:Main:AutoAllyingTextRandomTeam=允许一个玩家在此，但不分配队伍。; Allow a player here, but don't assign a team.
```

even an AI can give you the suggestion that sounds fluent

```suggestion
Client:Main:AutoAllyingTextRandomTeam=允许玩家加入但不分配队伍。; Allow a player here, but don't assign a team.
```